demisto_sdk/commands/init/initiator.py

```diff
@@ -388,28 +496,103 @@ def pack_init(self) -> bool:
         ).lower()
         if create_integration in ["y", "yes"]:
-            is_same_category = str(
-                input(
-                    "\nDo you want to set the integration category as you defined in the pack "
-                    "metadata? Y/N "
-                )
-            ).lower()
+            if not self.xsiam:
+                is_same_category = str(
+                    input(
+                        "\nDo you want to set the integration category as you defined in the pack "
+                        "metadata? Y/N "
+                    )
+                ).lower()
 
-            integration_category = (
-                self.category if is_same_category in ["y", "yes"] else ""
-            )
+                integration_category = (
+                    self.category if is_same_category in ["y", "yes"] else ""
+                )
+            else:
+                integration_category = INTEGRATION_CATEGORIES[0]
             integration_init = Initiator(
                 output=os.path.join(self.full_output_path, "Integrations"),
                 integration=True,
                 common_server=self.common_server,
                 demisto_mock=self.demisto_mock,
                 template=self.template,
                 category=integration_category,
+                xsiam=self.xsiam,
             )
             return integration_init.init()
 
         return True
 
+    def modeling_rules_init(self) -> bool:
+        """Creates a modeling rules directory tree.
+
+        Returns:
+            bool. Returns True if the  modeling rule was created successfully and False otherwise
+        """
+        self.full_output_path = os.path.join(self.output, self.dir_name)
+        self.get_created_dir_name(created_object="modeling rules")
+        modeling_rules_template_files = self.get_template_files()
+        if not self.get_remote_templates(
+            modeling_rules_template_files, dir=MODELING_RULES_DIR
+        ):
+            local_template_path = os.path.normpath(
+                os.path.join(
+                    __file__,
+                    "..",
+                    "templates",
+                    self.HELLO_WORLD_EVENT_COLLECTOR_INTEGRATION,
+                    "ModelingRules",
+                    self.template,
+                )
+            )
+            copy_tree(str(local_template_path), self.full_output_path)
+            self.create_files_from_local_template_or_empty(
+                modeling_rules_template_files, local_template_path
+            )
+
+        if self.id != self.template:
+            # note rename does not work on the yml file - that is done in the yml_reformatting function.
+            self.rename(current_suffix=self.template)
+            self.modeling_or_parsing_rules_yml_reformatting(
+                current_suffix=self.dir_name
+            )
+
+        return True
+
+    def parsing_rules_init(self) -> bool:
```

Comment:
This is basically the same function as modeling_rules_init
Let's make it general. pass to it the type and then change the values accordingly